src/components/menu/menu.scss

```diff
@@ -28,19 +28,4 @@ $menu-item-background-color: #ebebeb;
 
```

Comment:
The changes to `menu.scss` and `portal.scss` in `fix(portal): give portal exact size of its parent` seem like they might belong together with the changes to `portal.tsx` and `menu-surface.tsx` in the first two commits.

Reply:
⚡